### PR TITLE
csv dates come as strings. Type-check to see what's going on and conv…

### PR DESCRIPTION
**Pull request by thanasions**

### CODE REVIEW

File: scripts/tfidf_reduce.py [hunk 1]
```diff
@@ -73,6 +74,8 @@ def extract_ngrams_from_docset(self, pick_method, verbose=True):
 
     def create_terms_count(self, df, dates_header):
         dates = df[dates_header]
+        if type(dates[0]) is str:
+            dates = pd.to_datetime(dates).tolist()
```
Comment (IanGrimstead) [inline scripts/tfidf_reduce.py]: Other option is to grab a dataframe as soon as it lands and then convert the column format - elsewhere in the code, everything is consistent and no-one needs to check formats

Comment (thanasions) [reply scripts/tfidf_reduce.py]: was thinking about it, let me check

File: scripts/tfidf_reduce.py [hunk 2]
```diff
@@ -73,6 +74,8 @@ def extract_ngrams_from_docset(self, pick_method, verbose=True):
 
     def create_terms_count(self, df, dates_header):
         dates = df[dates_header]
+        if type(dates[0]) is str:
```
Comment (IanGrimstead) [inline scripts/tfidf_reduce.py]:
```suggestion
```

File: scripts/tfidf_reduce.py (hunk 1 above)
Comment (IanGrimstead) [inline scripts/tfidf_reduce.py]:
```suggestion
```

File: scripts/tfidf_reduce.py (hunk 2 above)
Comment (IanGrimstead) [reply scripts/tfidf_reduce.py]: Conversion now covered in utils

File: scripts/tfidf_reduce.py (hunk 1 above)
Comment (IanGrimstead) [reply scripts/tfidf_reduce.py]: Conversion now covered in utils

**Pull request merged**